.github/workflows/update-semconv-integration-branch.yml

```diff
@@ -86,7 +86,9 @@ jobs:
 
           sed -i "s/^\tsemconv-pin = .*/\tsemconv-pin = $commit_desc/" .gitmodules
 
-          if ! git diff-index --quiet HEAD; then
+          git add content-modules/semantic-conventions .gitmodules
```

Comment:
How about:

```suggestion
          git add -A
```

Reply:
Hey @trask - you marked this as resolved but I'm not seeing the change yet. Is the update still pending?

Reply:
I thought I hit the commit suggestion button, but clearly not 🤔